packages/shared/src/components/feeds/FeedSettings/sections/FeedSettingsGeneralSection.tsx

```diff
@@ -36,111 +49,179 @@ export const FeedSettingsGeneralSection = (): ReactElement => {
             type={TypographyType.Callout}
             color={TypographyColor.Tertiary}
           >
-            Choose a name that reflects the focus of your feed.
+            {isMainFeed ? (
+              <span>
+                Want a custom feed name? You can always{' '}
```

Comment:
Should this copy be exposed to everyone?
Just thinking about non-enrolled members.

Reply:
updated so it only shows when you are plus